docs/examples/azure/gitversion/execute/usage-examples.md

```diff
@@ -4,6 +4,13 @@ Find out how to use the **gitversion/execute** task using the examples below.
 
 > The examples use the latest _0.x_ version of the GitVersion Execute task.  It is recommended to use the latest released version in your own pipelines.
 
+Note that if the pipeline is setup to use a shallow git fetch mode the GitVersion Execute task will fail. It is required to use fetchDepth of 0 like so:
+
+```yaml
+- checkout: self
+  fetchDepth: 1
```

Comment:
You have `required to use fetchDepth of 0` in your comment, but `fetchDepth: 1` in your example.

There is an existing note in the [Github usage example](https://github.com/GitTools/actions/blob/main/docs/examples/github/gitversion/execute/usage-examples.md), perhaps it can be copied from there?

Reply:
I agree with that, we need to add the note as well, would be nice to keep them in sync as they share the same _interface_ but different implementations.